client/state/data-layer/wpcom/sites/blog-stickers/index.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External Dependencies
+ */
+import { translate } from 'i18n-calypso';
+import { isArray } from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import { SITES_BLOG_STICKER_LIST } from 'state/action-types';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { errorNotice } from 'state/notices/actions';
+import { addBlogStickerHandler } from 'state/data-layer/wpcom/sites/blog-stickers/add';
+import { mergeHandlers } from 'state/action-watchers/utils';
+
+export function requestBlogStickerList( { dispatch }, action ) {
+	dispatch(
+		http( {
+			method: 'GET',
+			path: `/sites/${ action.payload.blogId }/blog-stickers`,
+			body: {}, // have to have an empty body to make wpcom-http happy
+			apiVersion: '1.1',
+			onSuccess: action,
+			onFailure: action,
+		} ),
+	);
+}
+
+export function receiveBlogStickerList( store, action, next, response ) {
+	// validate that it worked
+	if ( ! response || ! isArray( response ) ) {
```

Comment:
All valid responses from this endpoint return an array:

<img width="614" alt="screen shot 2017-06-15 at 17 42 21" src="https://user-images.githubusercontent.com/17325/27192130-14af7364-51f2-11e7-8796-7157f53b9a75.png">